x-pack/plugins/alerting/server/lib/alert_summary_from_event_log.ts

```diff
@@ -106,6 +87,37 @@ export function alertSummaryFromEventLog(params: AlertSummaryFromEventLogParams)
     }
   }
 
+  for (const event of executionEvents.reverse()) {
+    const timeStamp = event?.['@timestamp'];
+    if (timeStamp === undefined) continue;
+    const action = event?.event?.action;
+
+    if (action === undefined) continue;
+    if (action !== EVENT_LOG_ACTIONS.execute) {
+      continue;
+    }
```

Comment:
I do not think that we need that since we have the filter `filter: 'event.provider: alerting AND event.action:execute'` around the executionEvents